go/client/cmd_log_send.go

```diff
@@ -130,32 +133,44 @@ func (c *CmdLogSend) GetUsage() libkb.Usage {
 	}
 }
 
-func (c *CmdLogSend) logFiles(status *fstatus) libkb.Logs {
+func (c *CmdLogSend) logFiles(status *fstatus) (entries []libkb.LogSendEntry) {
 	logDir := c.G().Env.GetLogDir()
-	installLogPath, err := GetInstallLogPath()
-	if err != nil {
-		c.G().Log.Errorf("Error in GetInstallLogPath: %s", err)
-		installLogPath = ""
-	}
+
 	if status != nil {
-		return libkb.Logs{
-			Desktop: status.Desktop.Log,
-			Kbfs:    status.KBFS.Log,
-			Service: status.Service.Log,
-			Updater: status.Updater.Log,
-			Start:   status.Start.Log,
-			Install: installLogPath,
+		entries = []libkb.LogSendEntry{
+			{"Desktop", status.Desktop.Log},
+			{"Kbfs", status.KBFS.Log},
+			{"Service", status.Service.Log},
+			{"Updater", status.Updater.Log},
+			{"Start", status.Start.Log},
```

Comment:
Personally, I'd prefer a smaller change in `log send` because

1. it's used a lot for debugging
2. we have no tests for it

But do you feel strongly that we should change it?